storage/database/shardedPersister_test.go

```diff
@@ -0,0 +1,398 @@
+package database
+
+import (
+	"crypto/rand"
+	"math/big"
+	"sync"
+	"testing"
+
+	"github.com/multiversx/mx-chain-go/storage"
+	"github.com/stretchr/testify/require"
+)
+
+func TestNewShardedPersister(t *testing.T) {
+	dir := t.TempDir()
+	db, err := createPersister(dir, "sharded")
+	require.Nil(t, err)
+
+	_ = db.Put([]byte("aaa"), []byte("aaaval"))
+	_ = db.Put([]byte("aab"), []byte("aabval"))
+	_ = db.Put([]byte("aac"), []byte("aacval"))
+
+	err = db.Close()
+	require.Nil(t, err)
+
+	db2, err := createPersister(dir, "sharded")
+	require.Nil(t, err)
+
+	_, err = db2.Get([]byte("aaa"))
+	require.Nil(t, err)
+
+	_, err = db2.Get([]byte("aab"))
+	require.Nil(t, err)
+
+	_, err = db2.Get([]byte("aac"))
+	require.Nil(t, err)
+}
+
+func TestNewPersister(t *testing.T) {
+	dir := t.TempDir()
+	db, err := NewLevelDB(dir, 2, 1000, 10)
+	require.Nil(t, err)
+	require.NotNil(t, db)
+}
+
+func BenchmarkPersisterOneKey1mil(b *testing.B) {
+	entries, keys := generateKeys(1000000)
```

Comment:
Extract 1000000 in a const. Also for 2mil, 4mil, etc.

Reply:
done